jib-core/src/main/java/com/google/cloud/tools/jib/ncache/CacheWrite.java

```diff
@@ -21,7 +21,7 @@
 import java.util.Optional;
 
 /** Represents layer data to write to the cache. <b>Implementations must be immutable.</b> */
-public interface CacheWrite {
+interface CacheWrite {
```

Comment:
It looks like CacheWrite isn't part of the API and may not need to be an interface? We don't seem to really allow customizing it and most creation events of CacheWrite are directly from calling DefaultCacheWrite under the hood.

I would think, given how we've implemented it here, that CacheWrite could just be what DefaultCacheWrite is now doing?

Reply:
Yep, I'll remove the interface.